downstream/modules/hub/proc-approve-collection.adoc

```diff
@@ -12,7 +12,7 @@ You can approve collections uploaded to individual namespaces for internal publi
 
 .Procedure
 
-. {MenuTopAE} *{MenuACAdminCollectionApproval}*.
+include::snippets/snip_menu-top-ae.adoc *{MenuACAdminCollectionApproval}*.
```

Comment:
Replace with 
```
include::snippets/snip_menu-top-ae.adoc[]
*{MenuACAdminCollectionApproval}*.
```
The `includes::` statement must be on a separate line, and must be followed by `[]`